features/filesystem/bd/ChainingBlockDevice.cpp

```diff
@@ -211,6 +211,11 @@ bd_size_t ChainingBlockDevice::get_erase_size() const
     return _erase_size;
 }
 
+bd_size_t ChainingBlockDevice::get_erase_size(bd_addr_t addr) const
+{
+    return _erase_size;
+}
+
```

Comment:
Oh, this implementation is going to be much more complicated. We need to lookup the erase size for the block inside the list of block devices.